modules/extension/xsd/xsd-core/src/test/java/org/geotools/xsd/StreamingParserTest.java

```diff
@@ -39,4 +40,16 @@ public void testParseXXE() throws Exception {
         // StreamingParser returns null if the parsing fails
         assertNull(parser.parse());
     }
+
+    @Test
+    public void testParseWithJavaMethod() throws Exception {
+        ByteArrayInputStream in = new ByteArrayInputStream("<mails></mails>".getBytes());
+        StreamingParser parser =
+                new StreamingParser(new MLConfiguration(), in, "java.lang.Thread.sleep(5000)");
```

Comment:
Might want to make this >5s.   It shouldn't actually ever take 5seconds so increasing it shouldnt cause a build time issue.  
The reason is that the build server is quite busy and sometimes really slows down.
Feel free to disregard this.